### PR TITLE
Remove warnings for K8s 1.31 and OCP 4.17.

### PR DESCRIPTION
**Pull request by donatwork**
# Description
Remove warnings from the script installer when on K8s 1.31 and OCP 4.17

# GitHub Issues
List the GitHub issues impacted by this PR:

| GitHub Issue # |
| -------------- |
| https://github.com/dell/csm/issues/1472 |
| https://github.com/dell/csm/issues/1473 |

# Checklist:

- [X] I have performed a self-review of my own code to ensure there are no formatting, vetting, linting, or security issues
- [X] I have verified that new and existing unit tests pass locally with my changes
- [X] I have not allowed coverage numbers to degenerate
- [ ] I have maintained at least 90% code coverage
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] Backward compatibility is not broken

# How Has This Been Tested?
Please describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration

- [X] Tested installer on K8s 1.31, no warnings about K8s 1.31
- [X] Tested installer on OCP 4.17, no warnings about OCP 4.17

### CODE REVIEW

**Review by santhoshatdell — APPROVED** [review]
Should we also update the min required version?

**Comment by lukeatdell** [comment]
> Should we also update the min required version?

blame history shows k8s min version hasn't been updated in a while, but it looks like ocp is regularly updated.

File: dell-csi-helm-installer/verify-csi-powerstore.sh [hunk 1]
```diff
@@ -10,8 +10,8 @@
 
 # verify-csi-powerstore method
 function verify-csi-powerstore() {
-  verify_k8s_versions "1.24" "1.30"
-  verify_openshift_versions "4.15" "4.16"
+  verify_k8s_versions "1.24" "1.31"
+  verify_openshift_versions "4.15" "4.17"
```
Comment (AkshaySainiDell) [inline dell-csi-helm-installer/verify-csi-powerstore.sh:14]:
```suggestion
  verify_openshift_versions "4.16" "4.17"
```

**Review by AkshaySainiDell — CHANGES_REQUESTED** [review]

File: dell-csi-helm-installer/verify-csi-powerstore.sh (hunk 1 above)
Comment (donatwork) [reply dell-csi-helm-installer/verify-csi-powerstore.sh]: Done.

**Comment by donatwork** [comment]
> Should we also update the min required version?

Min versions confirmed and updated.

**Review by santhoshatdell — APPROVED** [review]

**Review by lukeatdell — APPROVED** [review]

**Pull request merged**